2/twemoji.js

```diff
@@ -459,7 +464,11 @@ var twemoji = (function (
             ret = ret.concat(' ', attrname, '="', escapeHTML(attrib[attrname]), '"');
           }
         }
-        ret = ret.concat('>');
+        if (options.selfClosing) {
```

Comment:
do we really need this? since there are no side effects whatsoever closing an `<img/>` even in jurassic `HTML`, what is the actual benefit of introducing a boolean flag when the library can already be smart enough and do that regardless?